importer/v8/importer.go

```diff
@@ -146,7 +162,7 @@ func (i *Importer) processDML(scanner *bufio.Scanner) {
 		if strings.HasPrefix(line, "#") {
 			continue
 		}
-		i.batchAccumulator(line)
+		i.batchAccumulator(line, start)
 	}
```

Comment:
Very minor, but I wouldn't bother with the variable. `i.batchAccumulator(line, time.Now())`. Purely cosmetic.